client/components/automattic-logo/index.tsx

```diff
@@ -1,15 +1,19 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { FunctionComponent } from 'react';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const AutomatticLogo = ( { className, size } ) => (
+interface Props {
+	className?: string;
+	size?: number;
+}
+
+const AutomatticLogo: FunctionComponent< Props > = ( { className = '', size = 96 } ) => (
```

Comment:
The default `""` seems silly.

```suggestion
const AutomatticLogo: FunctionComponent< Props > = ( { className, size = 96 } ) => (
```